engine/src/test/java/io/camunda/zeebe/engine/dmn/DecisionEvaluationTest.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.engine.dmn;
```

Comment:
🔧 Let's move this into the `processing` package. We could have ` dmn` package in there with this test.